apps/src/netsim/NetSimConnection.js

```diff
@@ -201,6 +201,16 @@ NetSimConnection.prototype.onMyNodeChange_= function () {
 };
 
 /**
+ * Detects when local client node is offline.  Responds by kicking the user
+ * out of the shard, which should update the interface and provide an
+ * opportunity to reconnect.
+ * @private
+ */
+NetSimConnection.prototype.onMyNodeLostConnection_ = function () {
+  this.disconnectFromShard();
+};
```

Comment:
Do we expect to do more her in the future? Could we just do setLostConnectionCallback(this.disconnectFromShard)?